### PR TITLE
[skip ci]: modify the pipeline scripts to accomodate cosmetic fixes

### PR DESCRIPTION
**Pull request by w3aman**
Signed-off-by: Aman Gupta <aman.gupta@mayadata.io>

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -63,138 +50,103 @@ KUBEADM-CLUSTER:
...
+    - ./openebs-nativek8s/pipelines/stages/3-functional/3F01-zfspv-custom-topology-verify
 
-ZFSPV-RAW-BLOCK-VOLUME:
+3F02-ZFSPV-RAW-BLOCK-VOLUME:
   extends: .func_test_template
   script:
-    - chmod 755 ./openebs-nativek8s/pipelines/stages/3-functional/ZFSPV-raw-block-volume/ZFSPV-raw-block-volume
-    - ./openebs-nativek8s/pipelines/stages/3-functional/ZFSPV-raw-block-volume/ZFSPV-raw-block-volume
+    - chmod 755 ./openebs-nativek8s/pipelines/stages/3-functional/3F02-ZFSPV-RAW-BLOCK-VOLUME
+    - ./openebs-nativek8s/pipelines/stages/3-functional/3F02-ZFSPV-RAW-BLOCK-VOLUME
```
Comment (nsathyaseelan) [inline .gitlab-ci.yml:65]: can you check the test path?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
 # e2e-nativek8s
-OpenEBS E2E Pipelines
+
+This repository's `release-branch` branch contains plateform execution scripts for each indivisual test which is meant to run for inducing chaos and testing zfs-localpv components. It includes several stages to perform different scenraios in end to end work-flow. To see the README's and test results of pipeline switch to the `release-branch` of this repository.
```
Comment (nsathyaseelan) [inline README.md:3]: typos 
** indivisual --> individual
** scenraios --> scenario

for readme status and test results need to move master or release branch?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,24 @@
...
+
+To see the CI/E2E summary of the gitlab pipelines for e2e-nativek8s project visit `openebs.ci` and switch to `k8s` tab under `stable releases` category.
+
+## Pipeline stages for running against release-branch
+
+1. cluster-setup
+2. provisioner
+3. functional
+4. chaos
```
Comment (nsathyaseelan) [inline README.md:12]: chaos stage is not there in this pipeline

File: openebs-nativek8s/pipelines/stages/2-provisioner/2P02-zfs-controller-high-availability [hunk 4]
```diff
@@ -53,6 +53,7 @@ testResult=$(kubectl get litmusresult ${test_name} --no-headers -o custom-column
 
 current_time=$(eval $time)
 bash openebs-nativek8s/utils/e2e-cr jobname:zfs-ctrl-high-availability jobphase:Completed end_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" test_result:$testResult
+python3 e2e-nativek8s/openebs-nativek8s/utils/result/result_update.py $job_id 1S01 1-cluster-setup "Provision the zfs-localpv driver" $testResult $pipeline_id $current_time $commit_id $gittoken
```
Comment (nsathyaseelan) [inline openebs-nativek8s/pipelines/stages/2-provisioner/2P02-zfs-controller-high-availability:56]: check the test case id

File: openebs-nativek8s/pipelines/stages/3-functional/3F02-zfspv-raw-block-volume [hunk 5]
```diff
@@ -61,6 +62,7 @@ testResult=$(kubectl get litmusresult ${test_name} --no-headers -o custom-column
 
 current_time=$(eval $time)
 bash openebs-nativek8s/utils/e2e-cr jobname:zfspv-raw-block-volume jobphase:Completed end_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" test_result:$testResult
+python3 e2e-nativek8s/openebs-nativek8s/utils/result/result_update.py $job_id 3F01 3-functional "verify the zfspv-raw-block-volume support" $testResult $pipeline_id $current_time $commit_id $gittoken
```
Comment (nsathyaseelan) [inline openebs-nativek8s/pipelines/stages/3-functional/3F02-zfspv-raw-block-volume:65]: check the test case id

File: README.md (hunk 3 above)
Comment (w3aman) [reply README.md:12]: As of now no test cases are there in chaos stage, but kept there for future reference

**Pull request merged**